### PR TITLE
Set up JavaScript & styling for extra rsvp fields

### PR DESCRIPTION
**Pull request by Danielle-P**

### CODE REVIEW

File: src/style.less [hunk 1]
```diff
@@ -6,12 +6,18 @@ header {
...
   justify-content: center;
   align-content: center;
-  min-height: 200px;
+  min-height: 300px;
   padding: 1.8rem 0.8rem 1.6rem;
   background-repeat: no-repeat;
   background-position: center;
   background-size: cover;
 
+  .shrink-content {
```
Comment (benadamstyles) [inline src/style.less]: Get rid!

File: src/style.less [hunk 2]
```diff
@@ -101,11 +108,54 @@ h3 .superscript {
...
+    border-radius: 1px;
+    background: lighten(@secondary, 3%);
+    background-image: url('./images/chevron-down.svg');
+    background-position: right 0.75rem top 50%, 0 0;
+    background-repeat: no-repeat;
+    color: @body;
+    font-family: @body-font;
+    font-size: 1rem;
+    -moz-appearance: none;
+    -webkit-appearance: none;
```
Comment (benadamstyles) [inline src/style.less:119]:
```suggestion
```

File: src/style.less [hunk 3]
```diff
@@ -101,11 +108,54 @@ h3 .superscript {
...
+    border: none;
+    border-radius: 1px;
+    background: lighten(@secondary, 3%);
+    background-image: url('./images/chevron-down.svg');
+    background-position: right 0.75rem top 50%, 0 0;
+    background-repeat: no-repeat;
+    color: @body;
+    font-family: @body-font;
+    font-size: 1rem;
+    -moz-appearance: none;
```
Comment (benadamstyles) [inline src/style.less:118]:
```suggestion
```

File: src/style.less [hunk 4]
```diff
@@ -101,11 +108,54 @@ h3 .superscript {
...
+    -webkit-appearance: none;
+    appearance: none;
+
+    &:focus {
+      outline: none;
```
Comment (benadamstyles) [inline src/style.less]: Put this back or add your own

File: src/style.less [hunk 5]
```diff
@@ -101,11 +108,54 @@ h3 .superscript {
...
+    }
+
+    option {
+      &:hover {
```
Comment (benadamstyles) [inline src/style.less]:
```suggestion
      &:hover, &:focus {
```

**Review by benadamstyles — CHANGES_REQUESTED** [review]

File: src/style.less (hunk 5 above)
Comment (Danielle-P) [reply src/style.less]: I don't think the option:hover, &:focus styling is needed at all

**Review by benadamstyles — APPROVED** [review]

**Pull request merged**